apps/dot-voting/app/components/VoteDetails/VotingResults.js

```diff
@@ -32,6 +32,8 @@ const VotingResults = ({ vote, voteWeights }) => {
           totalSupport={totalSupport}
           color={`${theme.accent}`}
           voteWeights={voteWeights}
+          balance={!vote.open ? balance : 0}
+          symbol={(!vote.open && vote.data.tokenSymbol) ? vote.data.tokenSymbol : ''}
```

Comment:
These props seem overloaded with meaning to me. What do I mean by that? Not only you are passing down the balance and the symbol to `VotingOptions`, but also you are encoding in these two props the information of whether to show the allocations or not. I think this is error-prone because for instance what happens if `vote.open` is true but `vote.data.tokenSymbol` is falsey? Does that mean show or not show the allocations?

Maybe it makes more sense to make these checks explicit by including a `showAllocations` props set to `true` if `vote.open` and `vote.data.type === 'allocation'` (and `vote.data.tokenSymbol` is not falsey? I'm not sure). That way we can condense the logic for showing / not showing allocations in a single place and stop overloading these props. You can just pass `balance` and `vote.data.tokenSymbol` without having to check them.